Library/Formula/gobject-introspection.rb

```diff
@@ -16,7 +16,9 @@ class GobjectIntrospection < Formula
 
   depends_on "pkg-config" => :run
   depends_on "glib"
+  depends_on "cairo"
   depends_on "libffi"
+  depends_on :python
```

Comment:
Tim is likely to wince at the idea of enforcing our Python on such a major package, but I'll let him comment. @tdsmith